atsign-foundation/at_pico_w
# feat: ivNonce added to send impl

**- What I did**
Created a random ivNonce generator and used it in at_talk notifications sender method

**- How I did it**
See [original CPython secrets.py lib](https://github.com/python/cpython/blob/3.11/Lib/secrets.py)
Also see [constraints for MicroPython](https://docs.micropython.org/en/latest/genrst/modules.html#random)

**- How to verify it**
1. Connect to secondary and send `monitor verb` to listen for notifications
2. Send notifications from at_pico_w client and see how they are listened by the monitor verb, and how each of them has a different ivNonce
3. Open another at_talk client (recipient Atsign) to make sure that the decryption using the custom ivNonce is going well
